core/templates/dev/head/exploration_editor/main_tab/StateEditor.js

```diff
@@ -37,6 +39,12 @@ oppia.controller('StateEditor', [
     $scope.isInteractionIdSet = false;
     $scope.isInteractionShown = false;
 
+    $scope.oppiaBlackImgUrl = UrlInterpolationService.getStaticImageUrl(
+      '/avatar/oppia_black_72px.png');
+
+    $scope.getInteractionThumbnailImageUrl = (
```

Comment:
I don't think this is in the right place, if you want to call it from the interaction chooser modal...